All issues fixed

scaryghosty opened the pull request:
I know my commits are all over the place but every issue is fixed. I'd propose making a new branch and cleaning things up as my solutions are not ideal, but they work. In my fork I'm probably going to do just this.

I'm also going to see how hard it would be to tackle timezones.

License and Readme will need to be adjusted for copyright (did I do it right?) and for preferences (I changed a few things in the Readme).

scaryghosty committed:
Fix for https://github.com/stvhwrd/Ticker/issues/8

scaryghosty committed:
added a user prompt to press q

scaryghosty committed:
revised prompt

scaryghosty committed:
revised prompt a second time to be more clear

scaryghosty committed:
cleaned up interface

scaryghosty committed:
REALLY cleaned up the interface for my command.

scaryghosty committed:
fix for github.com/stvhwrd/Ticker/issues/8

scaryghosty committed:
ticker2 requires super user; ticker does not

scaryghosty committed:
updated requirements.txt

scaryghosty committed:
updated README.md

scaryghosty committed:
removed ticker2.py

scaryghosty committed:
updated README.md

scaryghosty committed:
fix for https://github.com/stvhwrd/Ticker/issues/2

scaryghosty committed:
updated README.md with python3 call

scaryghosty committed:
removed keyboard from requirements.txt

scaryghosty committed:
cleaned up network error; quit is now just enter

scaryghosty committed:
updated README.md to reflect functionality changes

scaryghosty committed:
disallowed echo so that no keys appear or do anything besides ENTER

scaryghosty committed:
Updated Codacy badge and copyright

scaryghosty committed:
Added my copyright to the list

scaryghosty committed:
minor wording update

scaryghosty committed:
added echo reenabling in all the places it needs to be

scaryghosty committed:
Merge branch 'master' of https://github.com/scaryghosty/Ticker
uh... why do i have to merge?

scaryghosty committed:
added timezone announcement to README.md

scaryghosty committed:
slight format change

scaryghosty committed:
updated copyright at end of README.md

scaryghosty committed:
format update

scaryghosty committed:
cleaned up network error interface drastically

scaryghosty committed:
Merge branch 'master' of https://github.com/scaryghosty/Ticker
?

scaryghosty committed:
updated copyright

scaryghosty committed:
fix for https://github.com/stvhwrd/Ticker/issues/11

scaryghosty committed:
Update ticker.py to make Windows compatible

scaryghosty committed:
Update README.md to reflect no Windows run

scaryghosty committed:
update ticker.py to re-disable echoing

scaryghosty committed:
update ticker.py to re-disable echoing (2)

stvhwrd commented:
Thanks for this, @scaryghosty! I'm getting a couple of errors on Windows, so I'll have a look tonight and follow up here.  Cheers 🍻

scaryghosty commented:
> Thanks for this, @scaryghosty! I'm getting a couple of errors on Windows, so I'll have a look tonight and follow up here. Cheers 🍻

I will attempt to address Windows compatibility issues in another branch on my fork.

scaryghosty committed:
Change quit key from 'ENTER' to 'Q'

added a custom function from https://www.jonwitts.co.uk/archives/896.
this function is getch() for linux. now Q quits instead of ENTER.
Updated readme to reflect these changes as well

scaryghosty committed:
Changed README.md to reflect my own mistake

ticker.py does not require 'python3 ticker.py'...
instead, it can be run with ./ticker.py after chmod u+x.

stvhwrd commented:
I don't have access to a Windows machine for the next while as I'm traveling so I'll merge this and create a placeholder issue to address the Windows problems.  Thanks again for all your work on this @scaryghosty.  Cheers :beers:
